src/chatdDb.h

```diff
@@ -284,7 +284,36 @@ class ChatdSqliteDb: public chatd::DbInterface
         if (idx != CHATD_IDX_INVALID)
             stmt << idx;
         stmt.stepMustHaveData("get peer msg count");
-        return stmt.intCol(0);
+        int32_t unReadCount = stmt.intCol(0);
+
+        sql = "select data from history where (chatid = ?1)"
+                "and (userid != ?2)"
```

Comment:
`userid` for management messages is the `COMMANDER`, isn't it? If so, we can actually search by `userid == COMMANDER`, but with the `type` and `chatid` should be enough (plus the `idx`)

Reply:
Alright, then we do have to keep the condition for the `userid != me` :) 
Thanks for checking